tests/FileSystemTests.cpp

```diff
@@ -650,7 +649,6 @@ TEST_CASE("FileSystemTests::GetFileInfo<FileStreamInfo>", "[filesystem]")
     wistd::unique_ptr<FILE_STREAM_INFO> streamInfo;
     auto hr = wil::GetFileInfoNoThrow<FileStreamInfo>(handle.get(), streamInfo);
     REQUIRE(hr == S_OK);
-#endif
 }
 
 TEST_CASE("FileSystemTests::QueryFullProcessImageNameW", "[filesystem]")

```

Comment:
nit: Not critical, but this could be pulled out of the `ifdef` for exception support